### PR TITLE
Urlchange

### PR DESCRIPTION
**Pull request by InnoraftxAmarta**
I have done some changes in URL search using Query parameters

### CODE REVIEW

File: index1.php [hunk 1]
```diff
@@ -5,6 +5,36 @@
...
+elseif($_GET['q']==3){
+    echo "<script>location.href='index3.php'</script>";
+
+}
+elseif($_GET['q']==4){
+    echo "<script>location.href='index4.php'</script>";
+
+}
+elseif($_GET['q']==5){
+    echo "<script>location.href='index5.php'</script>";
```
Comment (kunal-sin) [inline index1.php]: Please include file here instead of adding script here.

File: index2.php [hunk 2]
```diff
@@ -8,6 +8,36 @@
...
+    echo "<script>location.href='index4.php'</script>";
+
+}
+elseif($_GET['q']==5){
+    echo "<script>location.href='index5.php'</script>";
+
+}
+elseif($_GET['q']==6){
+    echo "<script>location.href='info.php'</script>";
+
```
Comment (kunal-sin) [inline index2.php]: Not sure why index2.php is required. Please make sure to have only one index file.

**Review by kunal-sin — CHANGES_REQUESTED** [review]
Please make sure to have only one index.php file and instead of adding script for query param, Please include the file.

File: index1.php (hunk 1 above)
Comment (InnoraftxAmarta) [reply index1.php]: added a landing page

File: index2.php (hunk 2 above)
Comment (InnoraftxAmarta) [reply index2.php]: this is 2nd page after login

**Comment by kunal-sin** [comment]
Please merge this PR @InnoraftxAmarta

**Pull request merged**